docs/reference/settings/ml-settings.asciidoc

```diff
@@ -109,3 +109,10 @@ cluster and the job is assigned to run on that node.
 IMPORTANT: This setting assumes some external process is capable of adding ML nodes
 to the cluster. This setting is only useful when used in conjunction with
 such an external process.
+
+`xpack.ml.process_connect_timeout` (<<cluster-update-settings,Dynamic>>)::
+Some {ml} processing is done by processes that run separately to the {es} JVM.
```

Comment:
It would be nice to have a brief introductory explanation here, akin to what we have in other timeout definitions. 
e.g. 
> The connect(ion?) timeout for {ml} processes that run separately from the {es} JVM. Defaults to `10s`. When such processes...